lib/lrama/lexer.rb

```diff
@@ -71,8 +71,7 @@ def lex_token
           # noop
         when @scanner.scan(/\/\*/)
           lex_comment
-        when @scanner.scan(/\/\//)
-          @scanner.scan_until(/\n/)
+        when @scanner.scan(/\/\/.*?\n/)
```

Comment:
This is changed by https://github.com/ruby/lrama/pull/245. Could you rebase (or merge) master branch and use current master implementation?

Reply:
Sorry for the delay. I updated this PR.

Reply:
No worries!